ia/src/process.py

```
@@ -2,8 +2,9 @@
 from models import iaParser
 from parser import ia_parser
 from utils import exit_error
+from message import *
```

Ruff (0.11.9)

`from message import *` used; unable to detect undefined names

(F403)